schema/omega-services.async.api.yaml

```diff
@@ -1,525 +1,415 @@
       message:
         oneOf:
-          - $ref: '#/components/messages/getEditSet'
-          - $ref: '#/components/messages/getEditSetRecord'
-          - $ref: '#/components/messages/updateEditSetRecord'
-
-  omega.editorial.web.application.instance.1:
-    description: The queue on which response messages will be sent.
+          - $ref: '#/components/messages/ListAssetLegalStatusSummary'
+          - $ref: '#/components/messages/ListAssetLegalStatusFull'
+  catalogue.editorial.web.application.instance.1:
+    description: >-
+      The queue on which response messages will be sent to instance 1 of the
+      Catalogue Editorial Web Application
     subscribe:
-      operationId: omega_editorial_web_application_instance_1
-      description: Response messages
+      operationId: catalogue_editorial_web_application_instance_1
+      description: >-
+        Response messages channel for instance 1 of the Catalogue Editorial Web
+        Application
       message:
         oneOf:
-          - $ref: '#/components/messages/editSet'
-          - $ref: '#/components/messages/editSetRecord'
-          - $ref: '#/components/messages/updateResponseStatus'
-          - $ref: '#/components/messages/errorResponse'
-
-  ### TODO(AR) request/response pattern
-  ### TODO(AR) - where to publish response messages
-  ### TODO(AR) - correlationId for responses (SQS+JMS)
-  ### TODO(AR) - auth on requests
-  ### TODO(AR) - error responses
-
-  ### TODO(AR) - is this language powerful enough to express real constraints and reuse - perhaps need XML Schema that is translated down into this?
-
+          - $ref: '#/components/messages/AssetLegalStatusSummaryList'
+          - $ref: '#/components/messages/AssetLegalStatusFullList'
 components:
   messages:
-    getEditSet:
-      messageId: OSGEES001
-      name: getEditSet
-      title: Get EditSet
-      summary: Retrieve an EditSet
-      # contentType: application/json
+    ListAssetLegalStatusSummary:
+      messageId: OSLISALS001
+      name: ListAssetLegalStatusSummary
+      title: List Asset Legal Status Summaries
+      description: List a summary description of each Legal Status applicable to any Asset
+      traits:
+        - $ref: '#/components/messageTraits/JMSMessage'
+        - $ref: '#/components/messageTraits/OMGMessage'
+        - $ref: '#/components/messageTraits/OMGRequest'
+        - $ref: '#/components/messageTraits/OMGAuthentication'
+    ListAssetLegalStatusFull:
+      messageId: OSLIFALS001
+      name: ListAssetLegalStatusFull
+      title: List Asset Legal Status
+      description: List a full description of each Legal Status applicable to any Asset
       traits:
-        - $ref: '#/components/messageTraits/serviceRequestHeaders'
+        - $ref: '#/components/messageTraits/JMSMessage'
+        - $ref: '#/components/messageTraits/OMGMessage'
+        - $ref: '#/components/messageTraits/OMGRequest'
+        - $ref: '#/components/messageTraits/OMGAuthentication'
```

Comment:
I'm a bit confused by this. Is this something we need right now? It seems to me we only need the legal status summaries for the edit record page.

Reply:
Do you mean the `OMGAuthentication`? If so, you can ignore it for the time being, I just decided to take care of it at the same time in the spec as it made sense.

Reply:
No, I was referring to the `ListAssetLegalStatusFull`. I don't know where we would need that? Are you anticipating it being wanted somewhere?

Reply:
Regards `ListAssetLegalStatusFull` - It may not be needed right now, but it seemed appropriate to add it at the same time to the spec whilst I was working in that area - I figured that the additional data it provides (definition, scope, context, etc.) could eventually be used in the UI for tooltips, alt-text, and/or contextual help.